packages/account/src/Sections/Security/LoginHistory/login-history-table-row.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Table, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TLoginHistoryItems } from 'Types';
+
+const LoginHistoryTableRow = ({ id, date, action, browser, ip, status }: TLoginHistoryItems) => {
+    return (
+        <Table.Row className='login-history__table__row' key={id}>
+            <Table.Cell>
```

Comment:
maybe we can create some array for cells and use .map() here? 
it is up to you :)

Reply:
I think using .map here is unnecessary since it will map the arrays from `login-history-content`. this component is just to align the object items to display the table